challenge.rb

```diff
@@ -1,26 +1,26 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  return input.map {|word| word[0].upcase + word[1..-1]}
```

Comment:
Would you believe [Ruby has a method to capitalize strings](http://ruby-doc.org/core-2.2.2/String.html#method-i-capitalize)?